test/scrabble_test.rb

```diff
@@ -4,9 +4,59 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
```

Comment:
The tests that you have look great, but don't forget to test your helper methods!!